src/app/campaigns/components/campaigns-list-item/campaigns-list-item.component.html

```diff
@@ -264,7 +264,6 @@ <h3 style="font-size: 24px" class="title" [ngClass]="{ 'gray-title': !campaign.h
           style="text-transform: uppercase"
         >
           {{ 'AllCampains.campaign_ends_on' | translate }}:
-          <img src="assets/Images/calender.svg" width="16px" />
```

Comment:
Why delete this line?

Reply:
It shows an extra calender icon that marwa requested to be deleted.
I found out that ligne to be the problem